packages/inter-protocol/src/econCommitteeCharter.js

```diff
@@ -179,6 +179,9 @@ export const start = async (zcf, privateArgs, baggage) => {
       },
       makeCharterMemberInvitation: () =>
         zcf.makeInvitation(charterMemberHandler, INVITATION_MAKERS_DESC),
+      shutdown: () => {
```

Comment:
adding shutdown methods to both contracts in 1 commit is awkward - I expect them to land in separate PRs, since the testing considerations are different.

Reply:
Good point. I will create two separate PRs along with the tests.